### PR TITLE
Update SignalR Sample

### PR DESCRIPTION
**Pull request by mikaelm12**
Making our sample look nicer. This is more similar to what we have in the samples repo.

### CODE REVIEW

File: src/SignalR/samples/SignalRSamples/wwwroot/hubs.html [hunk 1]
```diff
@@ -83,6 +84,13 @@ <h4>Group Actions</h4>
 <script src="lib/signalr/signalr-protocol-msgpack.js"></script>
 <script src="utils.js"></script>
 <script>
+    var name = prompt("Please enter your name", "D'Artagnan");
```
Comment (halter73) [inline src/SignalR/samples/SignalRSamples/wwwroot/hubs.html]: Can we just have a normal text input box with a default value of "Anonymous" or "D'Artagnan" or whatever for the user name? That way it doesn't block the entire browser UI.

File: src/SignalR/samples/SignalRSamples/Hubs/Chat.cs [hunk 2]
```diff
@@ -9,58 +9,48 @@ namespace SignalRSamples.Hubs
...
+        public Task Send(string name, string message)
         {
-            await Clients.All.SendAsync("Send", $"{Context.ConnectionId} joined");
+            return Clients.All.SendAsync("Send", $"{name}: {message}");
         }
 
-        public override async Task OnDisconnectedAsync(Exception ex)
+        public Task SendToOthers(string name, string message)
         {
-            await Clients.Others.SendAsync("Send", $"{Context.ConnectionId} left");
```
Comment (halter73) [inline src/SignalR/samples/SignalRSamples/Hubs/Chat.cs]: Can we include the user name in the query string or something so we can keep the user joined/left messages?

File: src/SignalR/samples/SignalRSamples/wwwroot/hubs.html (hunk 1 above)
Comment (anurse) [reply src/SignalR/samples/SignalRSamples/wwwroot/hubs.html]: Agreed. We had the `prompt` in other places where we connected on page load. If we have a "Connect" gesture we can use a standard text box. I would also suggest having the "Connect" button check for a non-empty user name and produce an error (and not connect) in that case.

**Review by halter73 — APPROVED** [review]

File: src/SignalR/samples/SignalRSamples/wwwroot/hubs.html [hunk 3]
```diff
@@ -27,6 +27,7 @@ <h1 id="head1">SignalR Hubs Sample</h1>
             <option value="hubT">Hub&lt;T&gt;</option>
         </select>
 
+        <input type="text" id="displayname" placeholder="Enter User Name" />
```
Comment (anurse) [inline src/SignalR/samples/SignalRSamples/wwwroot/hubs.html]: I miss `D'Artagnan` 😄

Comment (mikaelm12) [reply src/SignalR/samples/SignalRSamples/wwwroot/hubs.html]: Same! ☹️

**Pull request merged**